procedures/katello/release.md.erb

```diff
@@ -27,6 +27,7 @@
 - [ ] Bump version:
   - [ ] Open a PR (or use cherry-pick PR) against the release branch which updates `lib/katello/version.rb` to <%= full_version %>
   - [ ] `git pull` to make sure you have the latest changes
+  - [ ] Update/add the CHANGELOG.md file: `GITHUB_ACCESS_TOKEN=<secret> ./tools changelog <config_file>`
```

Comment:
Isn't it predictable which config file this should be? On line 21 we already have this:
```suggestion
  - [ ] Update/add the CHANGELOG.md file: `GITHUB_ACCESS_TOKEN=<secret> ./tools changelog configs/katello/<%= short_version %>.yaml`
```